src/modules/formBuilder/components/itemEditor/conditionals/EnableWhenCondition.tsx

```diff
@@ -0,0 +1,300 @@
+    groupCode: !!state?.answerGroupCode,
+  });
+  return (
+    <Stack>
+      <Grid container gap={2}>
+        {/* COLUMN 1: Select the dependent source */}
+        <Grid item xs={4}>
+          <Controller
+            name={`${enableWhenPath}.${index}.question`}
+            control={control}
+            disabled={advanced.localConstant}
+            rules={{ required: 'Dependent Question is required' }}
+            render={({
+              field: { ref, disabled, ...field },
+              fieldState: { error },
+            }) => (
+              <SelectOption
+                label='Dependent Question'
+                options={itemPickList}
+                sx={disabled ? { display: 'none' } : undefined}
+                {...field}
+                textInputProps={{
+                  helperText:
+                    error?.message ||
+                    "Question who's response will determine whether the condition is met",
+                  error: !!error,
+                  inputRef: ref,
+                }}
+              />
+            )}
+          />
+          <Controller
+            name={`${enableWhenPath}.${index}.localConstant`}
+            control={control}
+            disabled={!advanced.localConstant}
+            rules={{
+              required: 'Local Constant or Dependent Question is required',
+            }}
+            render={({
+              field: { ref, disabled, ...field },
+              fieldState: { error },
+            }) => (
+              <TextInput
+                // FIXME should be a dropdown of available Local Constants for this Role
+                label='Local Constant'
+                helperText={
+                  error?.message ||
+                  "Local constant who's value will determine whether the condition is met"
+                }
+                sx={disabled ? { display: 'none' } : undefined}
+                error={!!error}
+                inputRef={ref}
+                {...field}
+              />
+            )}
+          />
+        </Grid>
+        {/* COLUMN 2: Select the comparison operator */}
+        {/* TOOD: the operator picklist should be conditional based on the type of the dependent item. For example, "Less Than" is only applicable to numbers */}
+        <Grid item xs={3}>
+          <Controller
+            name={`${enableWhenPath}.${index}.operator`}
+            control={control}
+            rules={{ required: 'Operator is required' }}
+            render={({ field: { ref, ...field }, fieldState: { error } }) => (
+              <SelectOption
+                label='Operator'
+                options={enableOperatorPickList}
+                {...field}
+                textInputProps={{
+                  helperText: error?.message,
+                  error: !!error,
+                  inputRef: ref,
+                }}
+              />
+            )}
+          />
+        </Grid>
+        {/* COLUMN 3: Select the comparison value */}
```

Comment:
This is verbose / brittle. "Column 3" has a bunch of different inputs, depending on the type, but only one of them should show up. Maybe look into an alternative approach where there is only one input element, and it saves to the correct field depending on the type.

Reply:
Requested review but I still want to look at this @martha so no rush